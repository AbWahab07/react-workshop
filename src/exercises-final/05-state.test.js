import React from 'react'
import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import StopWatch from './05-state'

test('renders', () => {
  const wrapper = mount(<StopWatch />)
  expect(mountToJson(wrapper)).toMatchSnapshot()
})

