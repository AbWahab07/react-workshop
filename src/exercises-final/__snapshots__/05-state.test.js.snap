exports[`test renders 1`] = `
<StopWatch>
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }>
    <label
      style={
        Object {
          "display": "block",
          "fontSize": "5em",
        }
      } />
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "#fff",
          "border": "1px solid #ccc",
          "fontSize": "2em",
          "margin": "0 5px",
          "padding": "15px",
          "width": "200px",
        }
      }>
      Start
    </button>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "#fff",
          "border": "1px solid #ccc",
          "fontSize": "2em",
          "margin": "0 5px",
          "padding": "15px",
          "width": "200px",
        }
      }>
      Clear
    </button>
  </div>
</StopWatch>
`;
